### DISCUSSION TITLE
Documentation is super confusing. Please ping me and I can help you make it make sense to non-experts

### DISCUSSION THREAD

**Discussion by evilgeniuscreative** [vuejs/pinia#2250] [Ideas]
I can't make heads or tails of this. Either in Options or Setup mode. It's like a Escher Russian Doll. Message me and let's go over it so people *actually* find it easy instead of it just saying it's easy?

**Comment by posva** [comment]
PRs to improve the docs are always welcome!
There is also a Discord server: https://discord.com/invite/HBherRA

**Reply by evilgeniuscreative** [reply]
> PRs to improve the docs are always welcome! There is also a Discord server: https://discord.com/invite/HBherRA

I am very happy to help improve the dogs in fact I have a long history of clarifying documentation since half my job has been user experience person for a long time. However, I would need assistance in 😆 understanding the docs and what they're trying to say so that I can clarify them. Is there a documentation channel on the discord, or at least someone I could team up with to explain things that I can't understand from the documentation so I thought I can clear it up for new users like myself in the future?
